assets/styles/elements.scss

```diff
@@ -765,3 +765,17 @@ details.footnote {
         }
     }
 }
+
+/* Adapted after: https://www.w3.org/WAI/tutorials/forms/labels/#note-on-hiding-elements
```

Comment:
You can use our existing class for that: https://github.com/datenanfragen/website/blob/93ecc4aa7ed846db3459e2e3f7be40e027d13ba9/assets/styles/framework.scss#L380-L389